fhir-server/liberty-config/server.xml

```diff
@@ -47,6 +47,7 @@
         Example trace specifications: 
         <logging traceSpecification="*=info:com.ibm.fhir.*=finer" traceFormat="BASIC"/>
         <logging traceSpecification="*=info:com.ibm.fhir.persistence.jdbc.dao.impl.*=fine" traceFormat="BASIC"/>
+        <logging traceSpecification="${TRACE_SPEC}" traceFileName="${TRACE_FILE}" traceFormat="${TRACE_FORMAT}" consoleLogLevel="${WLP_LOGGING_CONSOLE_LOGLEVEL}"/>
```

Comment:
line  58?

Reply:
Yes...important to keep that in case you overwrite it temporarily then want to set it back to what it was originally.

Reply:
Right, I think that is the outdated mode of operating. we really want folks to set the environment variable.